internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -136,6 +137,145 @@ func (t *CreateObjectTest) WrappedSucceeds() {
 	ExpectEq(obj, o)
 }
 
+////////////////////////////////////////////////////////////////////////
+// CreateObject
+////////////////////////////////////////////////////////////////////////
+
+type CreateObjectChunkWriterTest struct {
+	fastStatBucketTest
+}
+
+func init() { RegisterTestSuite(&CreateObjectChunkWriterTest{}) }
+
+func (t *CreateObjectChunkWriterTest) CallsWrapped() {
```

Comment:
are you verifying the request parameters here. can you name the method accordingly

Reply:
This test is validating that the wrapped bucket is called with the expected parameters. Let me know if you still think we should rename the test.